### DISCUSSION TITLE
Probleme d'affichage de mon site

### DISCUSSION THREAD

**Discussion by Alios5** [vuejs/vitepress#3943] [Q&A]
Excuse me I need help for the deployment of my site on Github it does not work well the CSS of the site is not taken care of I do not know how to regulate I followed the documentation vitepress as I could but I think that I made an hour whatever by

**Comment by brc-dd** [comment]
Have you configured `base` properly in your .vitepress/config.ts?

**Reply by Alios5** [reply]
Yes, it works now, but the home page images don't load.
![image](https://github.com/vuejs/vitepress/assets/89401184/28ba4c93-d796-471a-ab36-35f67e56fc77)

**Comment by Alios5** [comment]
:
`---
# https://vitepress.dev/reference/default-theme-home-page
layout: home

hero:
  name: "RT Sheet Editor" 
  tagline: "Rhyrhm Sheet Editor is a tool designed to simplify the creation on rhythm game." 
  image:
    src: ./Logo.png
    alt: VitePress
  actions:
    - theme: brand
      text: Get Started
      link: /Getting-stardet
    - theme: alt
      text: Download
      link: //epic-alios.itch.io/rt-sheet-editor

features:
  - icon:
      src: ./assets/brain.png
    title: Intuitive interface
    details: Our user-friendly interface makes it easy for users to load their music and add rhythm events with precision.

  - icon: 
      src : ./assets/Custom.png
    title: Customization
    details: Create unique rhythm sets by adding elements such as beats, notes and obstacles, with total control over timing and duration. 

  - icon:
      src : ./assets/export.png
    title: Export to JSON
    details: Export your event data as a JSON file for seamless integration into your rhythm game.

  - icon:
      src : ./assets/Universal.png
    title: Universal compatibility
    details: The fact that it exports a JSON file means it's compatible with a variety of game engines and platforms, offering maximum flexibility in game development.
`

**Reply by brc-dd** [reply]
Those icons should be placed in public directory and referenced using leading slash, eg. `/brain.png` . Assets referenced from frontmatter are not automatically copied to build.

**Reply by Alios5** [reply]
it still doesn't work

https://github.com/vuejs/vitepress/assets/89401184/edf8787f-59e8-45fe-95dd-277ce479806f

**Reply by brc-dd** [reply]
Those are not in public directory. Create docs/public/ and move them there.

**Reply by Alios5** [reply]
ok it work now thanks 
![image](https://github.com/vuejs/vitepress/assets/89401184/813dbd81-e66a-4a6f-88a1-2fd42dac0ee9)
